lib/modules/datasource/docker/index.ts

```diff
@@ -901,7 +901,7 @@ export class DockerDatasource extends Datasource {
       // These follow the format of {{registryHost}}{{jFrogRepository}}/library/{{dockerRepository}}
       if (
         (err.statusCode === 404 || err.message === PAGE_NOT_FOUND_ERROR) &&
-        is.string(err.headers['x-jfrog-version']) &&
+        isArtifactoryServer(err) &&
```

Comment:
In this case is `err` compatible with `res: HttpResponse` ?

Reply:
no. needs to be err.response or something like that

Reply:
it was compatible in this case, however changed it to `err.response` to be clearer